authors/apps/authentication/views.py

```diff
@@ -243,21 +243,28 @@ def create(self, request, *args, **kwargs):
             return Response({"error": "The email is already registered, please try another one"},
                             status=status.HTTP_400_BAD_REQUEST)
 
-        if user and user.is_active:
+        if user:
```

Comment:
Why did you remove the user.is_active check?
Shouldnt this be restricted to users that have actually activated their accounts?

Reply:
Alright. That makes sense